README.md

```diff
@@ -66,7 +66,7 @@ hzcloud starter-cluster create \
   --name=mycluster \
   --region=us-west-2 \
   --total-memory=0.2 \
-  --hazelcast-version=4.0
+  --hazelcast-version=5.0
```

Comment:
The current version is 5.0.2-2, so better to have `--hazelcast-version=5.0.2-2`. In the future, we will improve hz version selection smarter on backend side.